src/components/modals/PlaceOrderModal.tsx

```diff
@@ -36,19 +36,18 @@ export const PlaceOrderModal: React.FC<PlaceOrderModalProps> = ({
           Order now
         </Button>
       </DialogTrigger>
-      <DialogContent className='md:w-9/10 md:max-h-dialog-content md:max-w-dialog-content h-full w-full max-w-full rounded-none sm:rounded-none md:flex md:gap-0 md:rounded-md md:border-0 md:p-0'>
-        <div className='md:w-image-container relative hidden md:block'>
+      <DialogContent className='md:w-9/10 h-full w-full max-w-full rounded-none sm:rounded-none md:flex md:max-h-dialog-content md:max-w-dialog-content md:gap-0 md:rounded-md md:border-0 md:p-0'>
+        <div className='relative hidden md:block md:w-image-container'>
           <Image
             src='/form-decoration.png'
             alt='Form decoration'
-            layout='fill'
-            objectFit='cover'
-            objectPosition='center'
+            fill
+            sizes='300px'
```

Comment:
replaced deprecated props with the new ones. Had to specify the `sizes` in px because `w-image-container` was not being read/transpiled/I don't know how to explain it ;)

Reply:
The current sizes value of '300px' doesn't make much sense when you're using fill. The fill prop tells the image to expand and fill its parent container, so a fixed size like '300px' won't be effective.

Instead, sizes attribute should define how much space the image should occupy relative to its parent container at different breakpoints. Since the image is only visible on medium screens and larger (md:block),  it's  could adjusted like :  

  sizes='(min-width: 768px) 33vw, 100vw'

Reply:
thanks! Fixed that